Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3104,4 +3110,43 @@ extension BrowserViewController {
             NavigationPath.handle(nav: path, with: self)
         }
     }
+    
+    func requestReviewIfNecessary() {
+        #if DEBUG
```

Comment:
Do we want these requests to show up in beta builds? Maybe we should use `!AppConstants.BuildChannel.isRelease` instead here?

Reply:
Fixed.